### ISSUE TITLE
"This wasn't helpful" wasn't helpful

### ISSUE THREAD

**Issue by Tahnan** [galacticpuzzlehunt/gph-site#34]
Hi there!  I've run into an issue as an end user of gph-site: not running a hunt, but playing in hunts run using it.  I hit this in the most recent GPH, I think, and then also in the current ECPH:

When a hint is answered, there's a button you can click that's labeled "This wasn't helpful".  Several times, I've gotten a hint that *was* helpful, it just wasn't helpful *enough*, or I had a followup or some sort, and the button as labeled made me think I shouldn't click it.  In fact, in the GPH, I used a second hint on the same puzzle when that happened.  But based on clicking it in the ECPH just now, it looks like it explicitly says that what you're asking is a follow-up (and the running team may ask you to use a new hint)--that is, you are supposed to use it in the "helpful, but maybe not helpful enough" case.

Would it make sense to change the default text on this button to something like "Ask a follow-up question"?

Thanks!

**Comment by cesium12** [comment]
Yeah, this has come up in GPH feedback. The original was coded in a hurry and we kept it by inertia in 2022 since no one had made enough of an effort to settle a better phrasing for the button (or explainer text). I still don't have a great idea myself, but I'm happy to take a PR with a concrete proposal (it's in [hints.html](https://github.com/galacticpuzzlehunt/gph-site/blob/master/puzzles/templates/hints.html)).
